hw/ip_templates/otp_ctrl/dv/env/otp_ctrl_scoreboard.sv.tpl

```diff
@@ -214,11 +217,27 @@ else:
             exp_lc_data.state = otp_lc_data[LcCountWidth +: LcStateWidth];
 
             // Token values are depend on secret partitions value.
-            exp_lc_data.test_unlock_token =
-                    {<<32 {otp_a[TestUnlockTokenOffset/4 +: TestUnlockTokenSize/4]}};
-            exp_lc_data.test_exit_token =
-                    {<<32 {otp_a[TestExitTokenOffset/4 +: TestExitTokenSize/4]}};
-            exp_lc_data.rma_token = {<<32 {otp_a[RmaTokenOffset/4 +: RmaTokenSize/4]}};
+            // exp_lc_data.test_unlock_token =
+            //         {<<32 {otp_a[TestUnlockTokenOffset/4 +: TestUnlockTokenSize/4]}};
+            // exp_lc_data.test_exit_token =
+            //         {<<32 {otp_a[TestExitTokenOffset/4 +: TestExitTokenSize/4]}};
+            // exp_lc_data.rma_token = {<<32 {otp_a[RmaTokenOffset/4 +: RmaTokenSize/4]}};
+
+
+            for (int unsigned chunk =0; chunk < (TestUnlockTokenSize/4); chunk++) begin
+              exp_lc_data.test_unlock_token[(chunk*TL_DW) +: TL_DW] =
```

Comment:
How is this different from the previous code? Streaming assignments are more compact and flexible, but may take some getting used to them.